ee/agent/flags/flag_controller.go

```diff
@@ -341,6 +341,15 @@ func (fc *FlagController) ControlRequestInterval() time.Duration {
 	).get(fc.getControlServerValue(keys.ControlRequestInterval))
 }
 
+func (fc *FlagController) SetAllowDt4aAcceleration(enabled bool) error {
+	return fc.setControlServerValue(keys.AllowDt4aAcceleration, boolToBytes(enabled))
+}
+func (fc *FlagController) AllowDt4aAcceleration() bool {
```

Comment:
I thought we determined that we didn't want to accelerate on every call to `/dt4a`, I this a "just incase minds change"?

Reply:
Updated!

```
	// AllowDt4aAcceleration enables acceleration via /v3/dt4a localserver endpoint. It is a test flag
	// for development use; it should ultimately be replaced by a call to a new /v3 endpoint that only
	// performs acceleration.
```

Let me know if you think a flag rename would be useful also.

Reply:
The comment is good, thank you!

Reply:
seph suggested a name update to `allow_overly_broad_dt4a_acceleration` so I've done that as well! Hopefully makes it a little clearer 🤞